_test/check-configlet-fmt.sh

```diff
@@ -9,6 +9,25 @@ configlet="${repo}/bin/configlet"
 c="${repo}/config.json"
```

Comment:
Because the newly added code does not use `$c`, `$m`, `$repo`, and `$configlet`, and I think it is better to define variables close to where they are used: I think all the newly added code should go above all four of those definitions.

The potential room for disagreement is if someone thinks it's better to define all variables at the top, but I think this only holds true for variables that we may want to change often. I can see `configlet` wanting to change, but nothing else.